packages/models-library/src/models_library/generics.py

```diff
@@ -63,12 +62,5 @@ class Envelope(GenericModel, Generic[DataT]):
     error: Any | None = None
 
     @classmethod
-    def parse_data(cls, obj):
+    def from_data(cls, obj: Any) -> "Envelope":
         return cls.parse_obj({"data": obj})
-
-    @validator("data", pre=True)
-    @classmethod
-    def empty_dict_is_none(cls, v):
-        if v == {}:
-            return None
-        return v
```

Comment:
I believe @wvangeit tried to remove this code at some point and concluded that it is needed. But perhaps you already handled the cases where the code depends on this?

Reply:
I modified some parts to make this work.